manifest/ocischema/manifest_test.go

```diff
@@ -87,6 +90,9 @@ func TestManifest(t *testing.T) {
 	if !reflect.DeepEqual(&unmarshalled, deserialized) {
 		t.Fatalf("manifests are different after unmarshaling: %v != %v", unmarshalled, *deserialized)
 	}
+	if deserialized.Annotations["hot"] != "potato" {
```

Comment:
My reading of the spec is also that it doesn't have any specification about what URLs are allowed, or whether URLs are required. Your patch is perfectly spec compliant.

But what I mean by "policy" is the policy that the administrator has configured for a server/deployment. If the administrator has configured that only registry.example.com URLs are allowed, or that no URLs are allowed at all, then you'd expect that to apply to both v2 manifests and OCI manifests. It's very surprising if the policy only applies to v2 manifests, and you can bypass it by pushing an OCI manifest.